Backend/src/database/s3_database.py

```diff
@@ -46,15 +46,19 @@ def get_course(self, course_code: str, term: str) -> Course | None:
         
         return self.convert_to_course(course_map)
 
-    def get_terms(self) -> List[str]:
+    def get_terms(self) -> dict:
         '''
-        Gets the list of terms in the database.
+        Gets the terms dict in the database.
         '''
-        return list(self.terms_courses_dict.keys())
+        terms_course_copy = self.terms_courses_dict.copy()
+        for term_dict in terms_course_copy:
+            terms_course_copy.get(term_dict, {}).pop("Classes", None)
```

Comment:
This line can be changed to simply:
term_dict.pop("Classes", None)

Reply:
In this case term_dict is actual the term ("Fall 2023") so doing term_dict.pop("Classes", None) would not work. That was unclear due to me using "term_dict" so I updated the code to use the "term" instead in the loop